MantidPlot/src/AxesDialog.cpp

```diff
@@ -2015,32 +2012,28 @@ void AxesDialog::updateGrid() {
  */
 bool AxesDialog::pressToGraph() {
   // Check if all tabs and axes are valid first
-  for (auto axisItr = m_Axis_list.begin(); axisItr != m_Axis_list.end();
-       axisItr++) {
-    if (!((*axisItr)->valid())) {
+  for (auto &axisItr : m_Axis_list) {
```

Comment:
Can we remove "Itr" from these variable names given that it is no longer an iterator.